docs/development/release.rst

```diff
@@ -90,9 +90,8 @@ Steps for the day to announce the release:
    (decide on a case by case basis, if it's relevant to the group of people):
 
     * https://groups.google.com/forum/#!forum/astropy-dev
-    * https://lists.nasa.gov/mailman/listinfo/open-gamma-ray-astro
-    * CTA DATA list (cta-wp-dm@cta-observatory.org)
-    * hess-analysis
+    * CTAO AS WG list (cta-wg-as [at] cta-observatory [dot] org)
+    * hess-forum list (hess-forum [at] lsw.uni-heidelberg [dot] de)
```

Comment:
Why not MAGIC as well now that it is an official s/w?

Reply:
If you have a contact mailing list, you can insert it directly ;)

Reply:
I just asked them. Would it be possible also to notify LST?

Reply:
For MAGIC we can use `magic_soft [at] mppmu.mpg [dot] de`